src/com/android/volley/Request.java

```diff
@@ -699,4 +704,17 @@ public void setJoined(boolean joined) {
     public void expireCache() {
         // no - op
     }
+
+    public boolean isExecuting() {
```

Comment:
in the current bug a request gets lost where it would be `!isExecuting` and `!isFinished`.